packages/@webex/plugin-meetings/src/multistream/mediaRequestManager.ts

```diff
@@ -123,12 +123,13 @@ export class MediaRequestManager {
 
   private getDegradedClientRequests(clientRequests: ClientRequestsMap) {
     const maxFsLimits = [
-      getMaxFs('best'),
-      getMaxFs('large'),
-      getMaxFs('medium'),
-      getMaxFs('small'),
-      getMaxFs('very small'),
-      getMaxFs('thumbnail'),
+      MAX_FS_VALUES['1080p'],
```

Comment:
`maxFsLimits` is not a lookup table, it's just an array of decreasing resolutions, previously when it was calling `getMaxFs()` it made sense to have the 2 separate entries for 'best' and 'large' as in theory `getMaxFs()` could return different maxFs values for them, but now that we're using `MAX_FS_VALUES` directly, I don't see any need to have 1080p duplicated in the first 2 entries